### DISCUSSION TITLE
Selection is erased after leaving a QSelect field via a click on another field

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#13161] [General - Components / Directives / etc] [answered]
I have the following QSelect in a form:

                <q-select
                  v-model="currentCase.Topic"
                  :options="filteredTopics"
                  class="text-bold col-4"
                  label="topic"
                  new-value-mode="add"
                  use-input
                  @filter="filterTopics"
                ></q-select>

This results in the following visual where I can type in the field and the possible entries are selected:

![image](https://user-images.githubusercontent.com/1345886/163116086-ec9706cf-efcf-4012-b01a-2131b3948d16.png)

The alternative is to type a new entry:

![image](https://user-images.githubusercontent.com/1345886/163116298-4254d57b-f232-4af0-891c-684a32b2c2df.png)

**My problem**: when typing a word (and possibly clicking on a suggestion), I have to either end the entry by pressing `Enter` or `Tab`. Clicking on another field erases the choice (the field is empty again).

I see that the [same behaviour](https://quasar.dev/vue-components/select#create-new-values) is present in the docs, so I believe it is not a bug. Is there a way to keep what was typed in when leaving the QSelect via a click on another field?

**Comment by metalsadman** [comment]
you can set `updateInputValue` method of QSelect, but i would appreciate if you make a codepen of what you are currently doing so we can work on it.

**Reply by wsw70** [reply]
Thank you. 
Sorry, I should have been more explicit with the fact that this is the behaviour of the docs. So the codepen is actually the one from https://quasar.dev/vue-components/select#using-menu-and-filtering: https://codepen.io/WoJWoJ/pen/KKZxZWo?editors=101. Unfortunately it does not work for me on neither Chrome nor FF (there are numerous errors in the console about blocked actions).

I am not sure there actually a solution - I found some other questions such as this one without a clear solution. I added

    @blur="updateInputValue(currentCase.Topic)"

(`currentCase.Topic` is the `v-model`), this leaves what was typed in in the field but the field description is conflicting with the display, and the value is not remembered

![image](https://user-images.githubusercontent.com/1345886/163345235-5de1233a-51c1-4e5d-958d-09d47fdb80e1.png)

**Comment by rstoenescu** [comment, marked as answer]
It's not a bug. Until you don't explicitly select an option, we have to clear out the input because the current selected value needs to be shown. Otherwise, it leads to unwanted behavior and confuses the user as to which is the current value.

**Reply by wsw70** [reply]
Thank you. I am not sure about the rationale, though: in contrast, nn input field works fine with the content being preserved between fields. It is just missing a dropdown where I could choose a value to pre-fill the field.

I managed to have something (https://github.com/quasarframework/quasar/discussions/13161#discussioncomment-2565269) but with your answer, it is now clear why it does not work.
